src/OpenSage.Game/Content/WindowLoader.cs

```diff
@@ -135,6 +135,7 @@ Control createControl()
                             SelectedItemBackgroundImage = imageLoader.CreateStretchableImage(wndWindow.EnabledDrawData, 1, 3, 2),
                             SelectedItemHoverBackgroundImage = imageLoader.CreateStretchableImage(wndWindow.HiliteDrawData, 1, 3, 2),
 
+                            Columns = wndWindow.ListBoxData.Columns,
```

Comment:
I’d prefer that column widths are resolved (ie if not specified, we split them equally) when creating the control. And that will make this extra Columns property redundant.